L1Trigger/TrackFindingTracklet/src/VMStubsTEMemory.cc

```diff
@@ -262,8 +278,15 @@ void VMStubsTEMemory::setbendtable(std::vector<bool> vmbendtable) {
 }
 
 void VMStubsTEMemory::writeVMBendTable() {
-  ofstream outvmbendcut;
-  outvmbendcut.open(getName() + "_vmbendcut.tab");
+  if (not std::filesystem::exists(settings_.tablePath())) {
+    system((string("mkdir -p ") + settings_.tablePath()).c_str());
```

Comment:
check if successful

Reply:
DONE
--  P.S. As this file/directory creation code appears in dozens of places, a future PR (almost ready) will move it to a single common function, to reduce duplicate code.